nodejs/awsx/ecs/cluster.ts

```diff
@@ -38,6 +38,11 @@ export class Cluster
      */
     public readonly securityGroups: x.ec2.SecurityGroup[];
 
+    /** @internal */
+    public readonly publicSubnetIds: pulumi.Output<string[]>;
+    /** @internal */
+    public readonly securityGroupIds: pulumi.Output<string[]>;
```

Comment:
Are these needed?  Can't we continue to compute these in callers?

Reply:
removed.